console/console-init/ui/src/components/MetadataListTable/MetadataListTable.tsx

```diff
@@ -68,7 +68,10 @@ const TableHeader: React.FC<IHeaderProps> = ({
   return (
     <DataListItem aria-labelledby={ariaLabelledby}>
       <DataListItemRow>
-        <DataListItemCells dataListCells={dataListCells()}></DataListItemCells>
+        <DataListItemCells
+          id="metadata-list-item-cells"
```

Comment:
If this is a row in the metadata list, it would be better named as 'metadata-list-row'. The idea is to name id as it appears in the UI, instead of using PF names.